Types/StatusCode.php

```diff
@@ -33,11 +33,11 @@ class StatusCode {
      * @var array
      */
     private static $expectedStatusCodes = [
-        'get'    => 200,
-        'put'    => 200,
-        'patch'  => 200,
-        'post'   => 201,
-        'delete' => 204
+        'get'    => [200, 203, 206, 404],
```

Comment:
This one is the only critical one we have to discuss. Everything else seems to be finde